scss/civicrm/dashboard/dashlets/_getting-started.scss

```diff
@@ -0,0 +1,19 @@
+#civicrm-getting-started {
+  #help {
+    margin: 0;
+  }
+
+  table {
+    border: 0;
+    margin: 7px 0;
+  }
+
+  td {
+    border: 0;
+    padding: 7px 15px;
```

Comment:
Cant we use existing variables? The margin/padding should be uniform in Shoreditch.

Reply:
This was the previous value we had and this value is also being used in multiple places without being defined in a variable. The weird thing is that is being used to pad tables, buttons, and ui tabs.

We can use Shoreditch defined variables for padding:
<table>
<tr>
<th>Original</th>
<th>Shoreditch Base</th>
<th>Shoreditch Large</th>
</tr>
<tr>
<td align="center">7px 15px</td>
<td align="center">6px 12px</td>
<td align="center">10px 16px</td>
<tr>
<td valign="top">
<img src="https://user-images.githubusercontent.com/1642119/111320160-1813a600-863d-11eb-8200-2eeb0028bd7a.png" width="400" />
</td>
<td valign="top">
<img src="https://user-images.githubusercontent.com/1642119/111321782-a3416b80-863e-11eb-9540-9251bf22a62b.png" width="400" />
</td>
<td valign="top">
<img src="https://user-images.githubusercontent.com/1642119/111320165-19dd6980-863d-11eb-9924-e26b1cea991e.png" width="400" />
</td>
</tr>
</table>

Which one should we choose?

Reply:
Lets keep it as it is then.